_includes/selector.html

```diff
@@ -513,115 +501,87 @@
                 }
+
                 var libraryToPkg = (pkg) => {
                     pkg = pkg.toLowerCase();
                     if (pkg === "cucim") return pkg;
                     return pkg + cuda_suffix;
                 }
-                var pkgs = this.active_packages.map(libraryToPkg).join(" ");
-                var isArm = this.active_arch === "arm";
-                var cupy_inst = "";
+                if (this.active_packages.length === 1 && this.active_packages[0] === "Choose Specific Packages") {
+                    return "Select at least one package.";
+                } else if (this.active_packages[0] === 'Standard') {
+                    var pkgs = this.additional_pip_packages.map(libraryToPkg).join(" ");
+                } else {
+                    var pkgs = this.active_packages
+                        .filter(pkg => pkg !== "Choose Specific Packages" && pkg !== "cuSignal")
+                        .map(libraryToPkg)
+                        .join(" ");                }
 
                 if (pkgs === "cucim") {
                     index_url = "";
                 }
 
-                return pip_install + pkgs + index_url + cupy_inst;
+                return pip_install + pkgs + index_url;
             },
             getDockerCmdHtml() {
-                var hasNotebooks = this.active_img_options.includes("notebooks");
-                var isDevel = this.active_img_options.includes("development");
+                var hasNotebooks = this.active_img_type === "Notebook";
                 var isNightly = this.active_release === "Nightly";
-                var isArm = this.active_arch === "arm";
                 var imgLoc = "";
-                if (this.active_img_loc === "NGC") imgLoc = "nvcr.io/nvidia/"
-                var imgVariant = "";
-                if (this.active_img_type === "core") imgVariant = "core";
-                var imgType = "base";
-                if (hasNotebooks) imgType = "runtime";
-                if (isDevel) imgType = "devel";
+                if (this.active_img_loc === "NGC") imgLoc = "nvcr.io/nvidia/";
+                var imgVariant = "Base";
+                if (this.active_img_type === "Notebooks") imgVariant = this.active_img_type;
+
                 var portOptions = (hasNotebooks ? ["8888:8888", "8787:8787", "8786:8786"] : [])
                     .map(opt => this.highlightFlag("-p") + " " + opt + " ").join("");
 
                 var imgRepo = [
-                    imgLoc + "rapidsai/rapidsai",
-                    imgVariant,
-                    (isDevel ? "dev" : ""),
+                    imgLoc + "rapidsai/" + imgVariant,
                     (isNightly ? "nightly" : ""),
-                    (isArm ? "arm64" : ""),
                 ].filter(Boolean).join("-");
 
                 var imgTag = [
-                    (isNightly ? this.getNightlyVersion() : this.getStableVersion()),
+                    (isNightly ? this.getNightlyVersion() + "a" : this.getStableVersion()),
```

Comment:
I can't find exactly where this happens, but when it is a nightly docker version, the only change is adding the `a`, so the code needs updating so that it doesn't output `rapidsai/base-nightly:23.10a-cuda11.8-py3.10`. Note the `-nightly` being appended to the repo name.

Reply:
Nice catch! Fixed in https://github.com/rapidsai/docs/pull/414/commits/7ac7dcd8fc00ae5038e1302dc00ed68c69c408bd